lua/telescope/builtin/__diagnostics.lua

```diff
@@ -69,7 +69,7 @@ local diagnostics_to_tbl = function(opts)
   opts.severity_limit = convert_diagnostic_type(severities, opts.severity_limit)
   opts.severity_bound = convert_diagnostic_type(severities, opts.severity_bound)
 
-  local diagnosis_opts = { severity = {}, namespace = opts.namespace }
+  local diagnosis_opts = { severity = nil, namespace = opts.namespace }
```

Comment:
we can't just do that. for 2 reasons, 
- we only wanna do that for nvim 0.10-dev and later 
- `diagnosis_opts.severity["min"] = opts.severity_limit` in line 84 will explode if we initialize severity to nil,

so i wouldnt touch this line, i'd rather think we should do this after we initialized `diagnosis_opts` with everything (i think line 90). so basically check if `#severity == 0` and nvim version > 0.9 set severity back to `nil`

Reply:
Thank for review. I add a check Neovim > 0.9.1. 89fb6ee

Reply:
probably unlikely this matters but maybe just `vim.version().minor > 9` in case neovim core releases a 0.9.2 without the api changes to `vim.diagnostic.get()`?

Reply:
@jamestrew sorry for late.

Yes, it is enough that we can check it by the version > 0.9.x. Fixed in 9dc3d90